db/seeds.rb

```diff
@@ -0,0 +1,46 @@
+require "bundler/setup" # require all the gems we'll be using for this app from the Gemfile. Obviates the need for `bundle exec`
+
+require "pg"
+require "active_record"
+require "pry"
+
+require_relative "../models/apartment"
+require_relative "../models/tenant"
+
+require_relative "../db/connection.rb"
```

Comment:
No need to reference `db` here since you're already in the `db` directory.

`require_relative "connection"` will do just fine.